js_modules/dagster-ui/packages/ui-core/src/search/AssetSearch.tsx

```diff
@@ -163,31 +210,39 @@ export const AssetSearch = () => {
   };
 
   return (
-    <SearchInputWrapper>
-      <SearchBox hasQueryString={!!queryString.length}>
-        <Icon name="search" color={Colors.accentGray()} size={20} />
-        <SearchInput
-          data-search-input="1"
-          autoFocus
-          spellCheck={false}
-          onChange={onChange}
-          onKeyDown={onKeyDown}
-          placeholder="Search assets"
-          type="text"
-          value={queryString}
-        />
-        {loading ? <Spinner purpose="body-text" /> : null}
-      </SearchBox>
-      <SearchResultsWrapper>
-        <SearchResults
-          highlight={highlight}
-          queryString={queryString}
-          results={renderedAssetResults}
-          filterResults={renderedFilterResults}
-          onClickResult={onClickResult}
-        />
-      </SearchResultsWrapper>
-    </SearchInputWrapper>
+    <div ref={searchRef}>
+      <ShortcutHandler onShortcut={hideSearchResults} shortcutFilter={shortcutFilter}>
+        <div />
```

Comment:
For some reason providing child components is required? Could change this in a future PR

Reply:
this looks like typing "/" into the search box will end up making it close. You can manually hook up the keydown listener to the `div` (`searchRef`) rather than rely on the keydown listener that ShortcutHandler makes.